### ISSUE TITLE
dash or plotly integration

### ISSUE THREAD

**Issue by vks2** [lovasoa/SQLpage#292]
hello
is there any plans to plotly or dash integration?
there'sv some js wrappers around

thanks much

**Comment by vks2** [comment]
datascientists afaik have a lot of plotly setups not easy to reassign to another drawing library. most of them is multichars that consists of 3-4 components, when you look at dataset from different point of views.

but if it's a great headache, and lot of work apex will be OK. guess we may somehow speed up plotting capabilities and tune up hoverability,

**Comment by lovasoa** [comment]
If you already have visualisations in python and are porting them to sqlpage, you will have to rewrite them anyway, even if the underlying library were the same, won't you ?

**Comment by vks2** [comment]
yes and no. datalayer should be written from scratch (like your code for x and y in charts examples), but with formatting plotly supports a powerful json parameterization that you inject into the engine. so one can build effective charts and do not depend on current code version or coding language. here's an example, hope you enjoy the idea behind: https://plotly.com/python-api-reference/generated/plotly.io.read_json.html indeed params could be very sophisticated 
![изображение](https://github.com/lovasoa/SQLpage/assets/49449761/d56aef96-ff5d-49d7-a604-e5227e916c06)

**Comment by vks2** [comment]
may I follow this example to integrate plotly? https://github.com/lovasoa/SQLpage/tree/main/examples/using%20react%20and%20other%20custom%20scripts%20and%20styles

**Comment by lovasoa** [comment]
Hello ! Yes, this example is still relevant !

**Comment by vks2** [comment]
looks pretty cool. will give it a try
